resources/web/wwi/MouseEvents.js

```diff
@@ -313,7 +320,8 @@ export default class MouseEvents {
     let orientation = WbWorld.instance.viewpoint.orientation;
     let position = WbWorld.instance.viewpoint.position;
 
-    let rotationCenter = new WbVector3((this.picker.coordinates.x / canvas.width) * 2 - 1, (this.picker.coordinates.y / canvas.height) * 2 - 1, this.picker.coordinates.z);
+    let rotationCenter = new WbVector3((this.picker.coordinates.x / canvas.width) * 2 - 1, (this.picker.coordinates.y /
+      canvas.height) * 2 - 1, this.picker.coordinates.z);
```

Comment:
It seems that it is better to break the line at higher level if possible
```suggestion
    let rotationCenter = new WbVector3((this.picker.coordinates.x / canvas.width) * 2 - 1,
      (this.picker.coordinates.y / canvas.height) * 2 - 1, this.picker.coordinates.z);
```